taca/utils/bioinfo_tab.py

```diff
@@ -76,8 +74,12 @@ def update_statusdb(run_dir):
                         obj={'run_id':run_id, 'project_id':project, 'flowcell': flowcell, 'lane': lane, 
                              'sample':sample, 'status':sample_status, 'values':{valueskey:{'user':'taca','sample_status':sample_status}} }
                         #If entry exists, append to existing
-                        if len(view[[project, flowcell, lane, sample]].rows) >= 1:
-                            remote_id = view[[project, flowcell, lane, sample]].rows[0].id
+                        #Special if case to handle lanes written as int
+                        if len(view[[project, run_id, int(lane), sample]].rows) >= 1:
+                            lane = int(lane)
```

Comment:
The first if makes sure the second int runs if lane happens to be an int. So remote_id and etc are still set even if lane is an int.